ext/extconf.rb

```diff
@@ -7,8 +7,55 @@
   $CPPFLAGS += $CXXFLAGS
 end
 
+brew = find_executable('brew')
+
+# Automatically use homebrew to discover thrift package if it is
+# available and not disabled via --disable-homebrew.
+use_homebrew = enable_config('homebrew', brew)
```

Comment:
nice that you can pass a default value into `enable_config` as the second arg - i'm learning tons of stuff about mkmf these days.